package.json

```
@@ -18,6 +18,7 @@
   "scripts": {
     "build": "rimraf dist && webpack -p",
     "prepack": "npm run build && npm run test",
+    "eslint:fix": "eslint --fix --ext .js src",
```

should be lint:fix
(eslint is a linter, not all linter are eslint, eg: in other repos, lint:fix starts eslint & prettier)

---

Please add lint:check too :)
```suggestion
    "lint:check": "eslint --ext .js,.vue src",
    "lint:fix": "eslint --fix --ext .js,.vue src",
```